### DISCUSSION TITLE
请问稍后再看可不可以增加观看进度显示

### DISCUSSION THREAD

**Discussion by xinblue** [cotaku/Bibi-Release#94] [General]
像官方uwp后来更新的稍后再看页一样，不再区分未观看和已观看（看到一半或者只看了1p的也会被划进已观看，清的时候会被一起清掉），而是显示观看进度，清除的时候只会清楚已看完的视频
非常感谢！

**Comment by cotaku** [comment]
增加了一个“正在观看”的列表，以及每个视频已观看的时长和总时长。
